workbench-core/swb-common-ui/src/common/labels.tsx

```diff
@@ -53,7 +53,8 @@ export const paginationLables: PaginationProps.Labels = {
   pageLabel: (pageNumber: number) => `Page ${pageNumber} of all pages`
 };
 
-export const headerLabels = {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const headerLabels: any = {
```

Comment:
can these labels be changed? if not we could probably remove the `any` and add  `as const` to the end of the object